README.md

```diff
@@ -1,39 +1,61 @@
-RaceCapture/Pro Firmware
-========================
+RaceCapturePro Firmware
+=====
 
-Firmware for RaceCapture/Pro data acqusition / control / telemetry system
+The premier open source telemetry system for your race vehicle.  RaceCapturePro is a
+device that gets installed into your race vehicle and beams telemetry about your
+vehicle and driving habits in real time.  Combined with
+[RaceCapture App](https://github.com/autosportlabs/RaceCapture_App) and
+[RaceCapture Live](https://race-capture.com/) a driver and their pit crew can
+get the information and extra edge needed to win.
 
-http://www.autosportlabs.net/RaceCapture
+<!--
+Inspiration
+-----
+WE SHOULD DO THIS.
+-->
 
-See the License.txt file for more information on licensing for this software.
+# Overview
 
-= Flashing =
+As of this writing there are two versions of RaceCapture hardware in the wild.
 
-WARNING: Updating firmware restores the default configuration on RaceCapture/Pro
-Be sure to save your configuration using the Race Analyzer software.
+## MK1
 
-================================
-Windows
-================================
+MK1 was the original RaceCapturePro unit.  Based on the AT91SAM7s chipset, it provided
+a solid platform for our initial release.  It allowd for up to a 100Hz channel sampling
+rate and up to 10Hz GPS.  It also supported CAN & OBD2, various I/O, an SD card, and
+a remote GPS mouse for high quality GPS reception.  It came with a separate telemetry
```

Comment:
Suggest: 

.... It also supported CAN and OBD-II (via external MCP2515 CAN controller), analog and digital sensors, an SD card ...